index.md

```diff
@@ -20,21 +20,6 @@ Learners are expected to have the following knowledge:
 
 ::: instructor
```

Comment:
Looks like this opening statement should also be removed? (or needs a closing if you like my comment below)

Reply:
I understand that this line now opens the "Breaks and Schedule" div.